tests/TestCase/Twig/Extension/StringsExtensionTest.php

```diff
@@ -98,23 +98,23 @@ public function testFilterTail()
         $input = 'Bob is 65 years old.';
         $callable = $this->getFilter('tail')->getCallable();
         $result = call_user_func_array($callable, [$input, 7]);
-        $this->assertSame('...old.', $result);
+        $this->assertSame('…s old.', $result);
     }
 
     public function testFilterTruncate()
     {
         $input = 'Bob is 65 years old.';
         $callable = $this->getFilter('truncate')->getCallable();
         $result = call_user_func_array($callable, [$input, 7]);
-        $this->assertSame('Bob ...', $result);
```

Comment:
They seem to have changed from `...` to `…` which is a https://www.compart.com/en/unicode/U+2026